### DISCUSSION TITLE
Is the project still maintained?

### DISCUSSION THREAD

**Discussion by itsjavi** [TanStack/virtual#570] [General] [answered]
Many open PRs, 4 stale branches out of 5, and no new commits since May, are there plans to keep maintaining this project?

I would like to contribute but collaborations seem also paralysed (13 open pull requests), so I feel kind of forced to fork it to fix things and release my own packages, which is something I wanted to avoid.

**Comment by GitMurf** [comment]
@tannerlinsley stuff is top notch across the board so I certainly doubt this project is being halted. My assumption was that it was just in a stable spot and he has many other things going on. But that all being said I am very interested to know what/if there are plans for the short term. Thanks for all your amazing work Tanner and team!

**Comment by GitMurf** [comment]
Also by the looks of the activity in related projects like https://github.com/TanStack/table/releases TanStack Tables (very active), I definitely would say the TanStack ecosystem isn't going anywhere :) One thought I've had for a while now is curious if Virtual Lists ever just gets merged with Tables and maybe something like that is in the works/plans 🤷‍♂️

**Comment by tannerlinsley** [comment, marked as answer]
Things are now up to date :)

I'd appreciate any validation to get v3 released as stable. :)

**Comment by GitMurf** [comment]
> Things are now up to date :)
> 
> 
> 
> I'd appreciate any validation to get v3 released as stable. :)

Do you just mean installing beta 56 and making sure it works fine and let you know if any issues arise the next few days?

**Reply by tannerlinsley** [reply]
Yep!

**Reply by jeg924** [reply]
@tannerlinsley What's the current status?

**Comment by itsjavi** [comment]
I still have the problem of package managers installing the alpha as the latest version.

About v0.56, I get this Error when trying to scroll on an empty list. The list was filtered in an autosuggest box, so the filtered element having the ref=virtualizer.measureElement was probably filtered out, and the lib complains.

![image](https://github.com/TanStack/virtual/assets/122741/41e3661b-f60c-4af3-b05e-0b798e061398)

Thank you for your time and dedication, and for keeping  the project alive!

**Comment by GitMurf** [comment]
@tannerlinsley should we be using .57 or .56? I see a tag for .57 but not a release. Not sure if that is intentional because it isn't ready to test? or if accidental?

.56 valid release: https://github.com/TanStack/virtual/releases/tag/v3.0.0-beta.56

.57 not a release: https://github.com/TanStack/virtual/releases/tag/v3.0.0-beta.57

**Comment by tannerlinsley** [comment]
Working on a release!
